applications/crossbar/src/api_resource.erl

```diff
@@ -111,6 +112,10 @@ rest_init(Req0, Opts) ->
             {'ok', cowboy_req:set_resp_header(<<"x-request-id">>, ReqId, Req9), Context3}
     end.
 
+-spec metrics() -> {non_neg_integer(), non_neg_integer()}.
+metrics() ->
+    {wh_util:bin_usage(), wh_util:mem_usage()}.
```

Comment:
Do we want to activate this for every request? are these calls without impact on perf?